doc/tor.md

```diff
@@ -211,3 +209,96 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- Some states have the ability to monitor and block use of the Tor network.
+  Recent examples include China and India. If you are in an environment that
+  does not permit direct Tor connections or the use of Tor bridges, then
+  considering the trade-offs, it may not be safe to use the Tor network.
+
+- For maximum privacy, it is preferable to disable listening instead of running an
+  onion service.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  Only connecting over Tor is more private when you combine it with no reachable IPv4/IPv6 address,
+  in particular if you want to broadcast transactions without them being correlatable with your IP.
+
+  On the other hand, if you only make random Tor connections, you're much more vulnerable to Sybil attacks
+  (as Tor addresses have 0 cost to create, an attacker can just flood the network with 1000s of apparent Tor
+  Bitcoin nodes, and have a high probability of receiving all outbound Tor connections a Tor-only node makes).
+  This is significantly less a concern with IPv4/IPv6 (especially with asmap) due to cost of getting IPs in many
+  networks. It's also less a concern if you have `-addnode` connections to trusted peers (even if they're onion addresses).
+
+  If everyone used `onlynet=onion` nobody on the onion bitcoin chain would be able to communicate with the clearnet chain.
+  It is essential that some nodes access both clearnet and Tor. If you only wish to give access to your node to other Tor users,
+  do not use it.
+
+## Setup Guide: Debian/Ubuntu
+
+Assuming you already have an installation of Bitcoin Core:
+
+**1. Install Tor:**
+
+To install Tor, run:
+
+```
+sudo apt install tor
+```
+
+2. Edit Tor configuration file:
```

Comment:
Since you made the first section `(1. Install Tor)` bold, as I suggested in this comment: https://github.com/bitcoin/bitcoin/pull/21157#discussion_r584419066. You have to make all other sections of this guide bold as I also suggested in the comment. That means lines `#250 (2. Edit Tor Configuration file:)`, `#265 (3. Add user to Tor group ("satoshi" in this example):`, `#271 (4. Restart Tor)`, `#279 (5. Edit Bitcoin configuration file)`, and `#290 (6. Confirmation)` must also go bold

```suggestion
**2. Edit Tor configuration file:**
```

Reply:
Done. Made changes in https://github.com/bitcoin/bitcoin/pull/21157/commits/8a32230482f612bb6bb79eb18c249152fd5daae3